seed/views/00501_team_capactiy_breakdown_view.js

```diff
@@ -0,0 +1,30 @@
+exports.seed = function (knex, Promise) {
+  var sql = `CREATE VIEW app.team_capacity_breakdown_view
+        WITH SCHEMABINDING
+        AS
+        SELECT
+            t.id AS id
+          , wo.id AS link_id 
+          , CONCAT(om.forename, ' ', om.surname) AS name
+          , omt.grade_code as grade_code
+          , w.total_cases
+          , wpc.total_points
+          , wpc.available_points
+          , wpc.reduction_hours
+          , wpc.cms_reduction_hours
+          , wpc.contracted_hours
+        FROM app.workload_points_calculations wpc
```

Comment:
This seems very similar to the overview view, with the addition of `cms_reduction_hours`. Any possibility of reusing those?

Reply:
I would prefer to keep it as separate.  In the next couple sprints we will be adding FDRs, SDRs etc to the capacity breakdown table.